flink-streaming-connectors/flink-connector-kinesis/src/main/java/org/apache/flink/streaming/connectors/kinesis/serialization/KinesisDeserializationSchema.java

```diff
@@ -52,5 +54,5 @@
 	 * @param nextElement the element to test for the end-of-stream signal
 	 * @return true if the element signals end of stream, false otherwise
 	 */
-	boolean isEndOfStream(T nextElement);
+	// TODO ADD SUPPORT FOR boolean isEndOfStream(T nextElement);
```

Comment:
Ah, I just noticed the JIRA for this. Let's leave it as it is then, and uncomment it for the fix on that JIRA.